### ISSUE TITLE
CVE-2022-3209

### ISSUE THREAD

**Issue by achifal** [nginxinc/docker-nginx#695]
```
+------------+-----------------------+--------------+-----------------+--------------------------------------------------------------+
| STATUS     | CVE SEVERITY          | PACKAGE NAME | PACKAGE VERSION | CVE DESCRIPTION                                              |
+------------+-----------------------+--------------+-----------------+--------------------------------------------------------------+
| Unapproved | Unknown CVE-2022-3209 | libxml2      | 2.9.14-r0       | https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2022-3209 |
+------------+-----------------------+--------------+-----------------+--------------------------------------------------------------+
```

**Comment by zops** [comment]
I also had the same problem.

**Comment by thresheek** [comment]
This is already fixed in Alpine repos, by libxml2 2.19.14-r1, but we'll need to wait for new `alpine` docker images for `nginx` ones to get rebuilt.

**Comment by KarthikBalaraman** [comment]
Libxml2 is added when we added nginx-module-xslt. Removing this module fixed the security vulnerability for me. Alpine latest (3.16.2) does not have this library at all. By looking at the docs I didn't need xslt module.

**Comment by vitoyucepi** [comment]
I think this issue was fixed in `nginx:1.22-alpine` and `nginx:1.23-alpine`.
@achifal, @thresheek could you check and close the issue?

Btw it's CVE-2022-2309.

**Comment by thresheek** [comment]
It's fixed in Alpine indeed.
